oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SQLDatabase.java

```diff
@@ -135,4 +136,12 @@
     @interface MultipleExtraColumn4AdditionalEntity {
         ExtraColumn4AdditionalEntity[] value();
     }
+
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface Sharding {
+        ShardingAlgorithm shardingAlgorithm();
+        String dsShardingColumn();
```

Comment:
In such case I'd rather write the full name, `dataSourceShardingColumn`, with IDE autocompletion it's the same key strokes but it is more clear